### PR TITLE
Improve win port build process

### PR DESCRIPTION
**Pull request by malirod**
**Update**
- use new names for the env vars. SDL_SQLITE_DIR for the sqlite and SDL_QT_DIR for the qt root.
- stop using env vars for the openssl. clone openssl sources into 3rd parties. Build it into "build" dir in the project root
- improve qt project generation. cmake will generate cbp file which is required for the Qt Creator. So now on fist run Qt Creator will ask for the build dir and toolchain. Set it build dir to the `<root>\build\sdl_win_qt_x86` and choose vs 2010 x86 toolchain.
  @pestOO, @anosach-luxoft, @nk0leg, please review

### CODE REVIEW

**Comment by malirod** [comment]
Seems it's `cbp` file is not enought for the Qt Creator.
We need `CMakeLists.txt.user` file in the project root. Otherwise the Qt Creator still asks for the build dir and toolchain on startup. After first run Qt Creator creates this file and don't ask again.

**Comment by malirod** [comment]
I've updated the PR description with the new info regarding Qt.

**Comment by anosach-luxoft** [comment]
@malirod I think it would be better to put openssl binaries into `3rd_party-static` instead of `3rd_party`

File: ms/openssl_setup.cmd [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  if %BUILD_TYPE%==x86 (
+    set OPENSSL_CONFIG_OPT=VC-WIN32 no-asm no-shared
+  ) else if %BUILD_TYPE%==x64 (
+    set OPENSSL_CONFIG_OPT=VC-WIN64A no-shared
+  ) else goto usage
+
+  pushd %OPENSSL_SRC_DIR%
+  call git.exe clean -fxd
+  call git.exe checkout origin/OpenSSL_1_0_2-stable
```
Comment (pestOO) [inline ms/openssl_setup.cmd:31]: May be git submodule?

Comment (malirod) [inline ms/openssl_setup.cmd:31]: I don't see the point. we are not going to do the contribution to the openssl. we will just read & build it. So the submodule is the overhead in this case. Plus I had problems with submodules in the past in other projects. So I am not sure that it worth to use them at all.

**Comment by malirod** [comment]
See 76d5da4

**Pull request merged**